Tools/clinic/clinic.py

```diff
@@ -6427,8 +6451,11 @@ def format_docstring_parameters(params: list[Parameter]) -> str:
     def format_docstring(self) -> str:
         assert self.function is not None
         f = self.function
-        if f.kind.new_or_init and not f.docstring:
-            # don't render a docstring at all, no signature, nothing.
+        if f.kind in {METHOD_INIT, METHOD_NEW, GETTER, SETTER} and not f.docstring:
```

Comment:
@erlend-aasland 
I inlined the previous code, IMO it will be more readable :)
(Not sure about cache effect)
https://github.com/python/cpython/blob/4afa7be32da32fac2a2bcde4b881db174e81240c/Tools/clinic/clinic.py#L2998-L2999